### PR TITLE
Fix y axis chart

### LINKED ISSUE #3: y_axis bounds do not change with temperature units.
As mentioned in #2 :

There is an issue with the y_axis bounds when dealing with F temps (they don't change).:
C temps  
!F temps

I believe it makes sense to change the bounds accordingly.

### PR DESCRIPTION
**Pull request by trinaldi**
This is most likely a hack, to be honest. However, both graphs are working. I tested running `yes` on my machine to heat it up and it seems to be working as expected - as far as a temperature chart works.

C Chart
F Chart

Left pane is the program, right pane is `yes` running.

This, most likely, fixes #3

### CODE REVIEW

**Review by olivesdad — APPROVED** [review]
LGTM thanks for contribution !

**Comment by trinaldi** [comment]
Thanks! I'm having a lot of fun (and frustration that comes with the job haha)

**Comment by trinaldi** [comment]
Conflicts resolved.

Fix #3

File: src/UI.rs [hunk 1]
```diff
@@ -155,6 +156,26 @@ pub fn ui(f: &mut Frame, app: &App) {
                 ),
         );
 
+    let chart_f = Chart::new(dataset)
```
Comment (olivesdad) [inline src/UI.rs:162]: ah this is a very interesting solution. instead of having to do the conversion we can just swap the axis labels! smart!

**Review by olivesdad — APPROVED** [review]
:partying_face:

**Pull request merged**